src/store/SwapMetaView.cc

```diff
@@ -84,7 +84,7 @@ Store::SwapMetaView::checkExpectedLength(const size_t expectedLength) const
 {
     if (rawLength != expectedLength)
         throw TextException(ToSBuf("Bad value length in a Store entry meta field expecting a ",
-                                   expectedLength, "-byte value: ", *this), Here());
+                                   expectedLength, "-byte value: ", this), Here());
```

Comment:
The proposed code change "prints" a _pointer_ to the view (i.e. view object's memory address) instead of printing the view itself. This change introduces a runtime bug AFAICT.